capstone/src/main/java/com/google/sps/Objects/Option.java

```diff
@@ -5,22 +5,22 @@
 public final class Option {
 
   private final String text;
-  private ArrayList<Integer> votes;
+  private ArrayList<String> userVotes;
```

Comment:
How are you handling making sure one user can only vote once?

Reply:
I want to make it so that people can vote on multiple options, kind of like how Facebook polling works

Reply:
Even so, with an array of strings per option, what's preventing one user from voting multiple times for one option?

Reply:
I was thinking that since Option has a list of userVotes, then I can search for the id of the current user and if the id is contained in that list, then I can set the checkmark as checked